### DISCUSSION TITLE
cypress 12 support

### DISCUSSION THREAD

**Discussion by AmirAmArA** [Shelex/cypress-allure-plugin#182] [General]
will there be a support for cypress 12 any time soon?

**Comment by AmirAmArA** [comment]
i keep getting peer dependency problems when trying to upgrade

**Reply by AmirAmArA** [reply]
@Shelex

**Comment by Shelex** [comment]
Hi @AmirAmArA 
I do not see any issues with cypress v12+, I have quickly tried v12.4.1 and plugin core functionality is working.
Could you please specify what peer dependency issues you get? This plugin is not using any dependencies related to cypress.

**Comment by Shelex** [comment]
Closing as no other input provided, and no other issues spotted in terms of using v12+
